js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1074,6 +1075,8 @@ var EditingController = modules.ViewController.inherit((function() {
             if(deferreds.length) {
                 that._saving = true;
 
+                dataSource && dataSource._changeLoadingCount(1);
```

Comment:
It will be a hack anyway. It's not loading but saving.
It would be helpful for us to understand why the panel is shown via DataSource and not by the Grid directly.